clio-server/src/main/scala/org/broadinstitute/clio/server/model/data.scala

```diff
@@ -2,9 +2,18 @@ package org.broadinstitute.clio.server.model
 
 import java.time.OffsetDateTime
 
+object ModelReadGroupLocation {
```

Comment:
This might need to go into the `transfer-model` project so the case classes there can use it.